arch/src/test/java/dk/ufst/arch/ComposeLocalStoreTest.kt

```diff
@@ -95,6 +91,40 @@ class ComposeLocalStoreTest {
         }
     }
 
+    @Test
+    fun `Test only one subscription per composable no matter how many state accesses`() {
```

Comment:
What happens if you have more than one `LocalStore`? For me it looks like this test will fail. Should the `GlobalStore` throw an exception if there are multiple subscriptions?

Reply:
Then you create more than one subscriber. 1 subscription per local store remembered. GlobalStore must allow multiple subscriptions since all created LocalStores accept one. You might be composing two views which each have their own subscription to GlobalStore

Reply:
Ok, but the test says `one subscription per composable` but if you add another LocalStore you have two subscriptions per composable. Or did I miss something?

Reply:
Oh, the test wants to verify that if you have one `LocalStore` you only have one subscriptions, right? So per composable doesn't matter?